department-of-veterans-affairs/va.gov-cms
# VACMS-1566: Improve clone default behavior.

## Description

See #1566 

## Testing done

Manual

## Screenshots

<img width="1299" alt="Entity_clone_settings___VA_gov_CMS" src="https://user-images.githubusercontent.com/643678/79927744-b0de7900-840e-11ea-836e-508041e116e3.png">


## QA steps

As user a content editor
Go to any node and clone it. 
You should only see paragraph entity references. 

As an admin, review http://va-gov-cms.lndo.site/admin/config/system/entity-clone
